catalogue_graph/tests/adapters/ebsco/test_transformer.py

```diff
@@ -456,6 +456,8 @@ def fake_bulk(client, actions, raise_on_error, stats_only):  # type: ignore[no-u
     monkeypatch.setattr("elasticsearch.helpers.bulk", fake_bulk)
 
     now = datetime.now()
+    now_iso = now.isoformat() + "Z"
```

Comment:
There are suggestions in application code files about UTC.  As this is freezing a time to create some test data, it probably doesn't matter much here, but we might want to add it for consistency (and just in case we change something else and the tests start mysteriously failing during BST)